docs/source/models/cause.rst

```diff
@@ -275,14 +273,155 @@ Birth Prevalence
+to as **incidence in the total population** (as estimated by the GBD study) 
+and **incidence in the susceptible** (or *at-risk*) **population.** These 
+measures are defined using the following key concepts:
+
+  **Person-time:** person-time is a measure of the number of individuals 
+  multiplied by the amount of time they individually occupy the population 
+  of interest. Notably, the population of interest varies depending on context 
+  and can be defined by age group, sex, location, time, disease status, etc.
+
+    For example, if one individual is occupies the population of interest for 
+    two years, they contribute two person-years. If another individual is in 
+    our population of interest for 6 months, they contribute 0.5 person-years.
+    Together, these two individuals contribute a total of 2.5 person-years.
+
+  **Susceptible or At-Risk Population:** the susceptible population, also 
+  referred to as the at-risk population, is defined as the population that *
+  does not* have the condition of interest; in other words, the susceptible 
+  population that is at risk of developing the condition. Notably, the number 
+  of individuals in this population will change overtime as the following 
+  events occur:
+
+     - Members of the at-risk population develop the condition and are no longer susceptible
+     - Members of the at-risk population die and are no longer susceptible 
+     - Individuals are born or age into the at-risk population and become susceptible
+     - Individuals age out of the at-risk population and are no longer susceptible
+     - Individuals with the condition recover from the condition and re-enter the at-risk population as susceptible (in the case of conditions with remission)
+
+**Total Population Incidence Rate** is estimated by the Global Burden of 
+Disease Study by estimating the number of incident cases that occur in one 
+year and scaling this value per 100,000 individuals of a specified population.
+
+.. math::
+
+  \frac{n_\text{incident cases}}{\text{person-time}_\text{total population}}
+
+Because the denominator of this measure is not specific to a particular cause 
+model state, it is **not** an appropriate data source for cause model 
+transition rates between states. 
+
+.. note:: 
+
+  GBD estimates of total population incidence rate require transformation 
+  prior to use as a cause model transition probability data source (see below 
+  for more detail).
+
+**Susceptible/At-Risk Population Incidence Rate** as discussed here is also 
+referred to as incidence density rate, person-time incidence rate, and in some 
+cases may simply be referred to as the incidence rate. It is defined as:
+
+.. math::
+
+  \frac{n_\text{incident cases}}{\text{person-time}_\text{susceptible population}}
+
+Because the denominator for the susceptible population incidence rate is 
+person-time in the at-risk population, it can represent the probability of a 
```

Comment:
I don't like the word "represent" here because the rate has dimensions of 1/time and is not the same thing as a probability, which is dimensionless (the rate is like a "probability density"). Rather, we can compute the probability if we know the rate (by integrating and then exponentiating). Here's a suggestion for different wording:

```suggestion
person-time in the at-risk population, this incidence rate can be used to compute the probability of a 
```

Reply:
Or perhaps, "after scaling by time, the incidence rate can represent the probability...". I think we just have to be careful about glossing over the rate/probability distinction because the rates can be bigger than 1, whereas probabilities cannot. The rate can only be interpreted as a probability after multiplying it by a sufficiently small time step. For sufficiently small time steps, rate*time becomes a good estimate of the probability, but how small of a time step you need to get a good approximation depends on how large the rate is.